tardis/base.py

```diff
@@ -49,6 +47,23 @@ def run_tardis(
         except TypeError:
             tardis_config = Configuration.from_config_dict(config)
 
+    if tardis_config["debug"]["logging_state"] or log_state:
+        if log_state:
+            logging_state(log_state)
+            if tardis_config["debug"]["logging_state"] and log_state:
+                print("Log_state & logging_level both specified")
+                print("Log_state will be used for Log Level Determination\n")
+        else:
+            logging_state(tardis_config["debug"]["logging_level"])
+    else:
+        logging_state(False)
+
+    if atom_data is not None:
```

Comment:
Why has this been moved?

Reply:
This has been moved because initially the order instantiated the atomic data before the configuration. For accessing values for the Logging level & Logging state, we need it through the configuration dictionary. Thus instantiating the config dictionary would allow for passing the required values to the `logging_state()` function. Calling `logging_state()` before the `atom_data` is also important as we are required to capture all the loggers for the simulation. I need to check whether vice versa (ie calling `logging_state` after `atom_data` does the job or not) is valid as well 😅